packages/manager/modules/billing/src/main/history/billing-main-history.constants.js

```diff
@@ -0,0 +1,5 @@
+export const HIDE_HTML_INVOICE_FOR_COUNTRIES = ['IN'];
```

Comment:
Please use feature flipping instead

Reply:
@marie-j, the logic should be implemented based on the user country and in this scenario, we cannot depend on subsidiary. Since feature-flipping does not support checking for the country, the implementation has been done in this manner.